Formula/gusto-lima.rb

```diff
@@ -0,0 +1,32 @@
+class GustoLima < Formula
+  desc "Gusto's opinionated colima profile"
+  homepage "https://github.com/abiosoft/colima/blob/main/README.md"
+  url "https://github.com/abiosoft/colima.git", tag: "v0.1.0"
+  license "MIT"
+  head "https://github.com/abiosoft/colima.git", branch: "main"
```

Comment:
This looks like it's from the upstream colima, right? I think one challenge here would be how to manage rolling out changes to this configuration, since it's always going to be v0.1.0 from this URL.

Reply:
Correct! We aren't using the upstream colima the url is required for Brew Formula. I have updated to use homebrew-gusto repo and updated the formula to use the version property instead of the repo tag so be can better manage new rollouts